decidim-elections/app/forms/decidim/elections/trustee_zone/trustee_form.rb

```diff
@@ -11,12 +11,17 @@ class TrusteeForm < Decidim::Form
         attribute :public_key, String
         validates :name, :public_key, presence: true
         validate :dont_change_data
+        validate :valid_name
 
         def dont_change_data
           errors.add :name, :cant_be_changed if trustee.name.present?
           errors.add :public_key, :cant_be_changed if trustee.public_key.present?
         end
 
+        def valid_name
+          errors.add :name, :is_taken if Decidim::Elections::Trustee.exists?(name: name)
```

Comment:
Should this be scoped to the current organization?

Reply:
Great question. 😀 I triple checked it and the answer is 'no'. One Decidim instance is connected to the `bulletin_board` and not each `organization`. So, that scope isn't relevant for the `trustee`. I updated the documentation to point this out. Hope, that's more understandable now. 😄

Reply:
Does this mean that I can't be the trustee of two different elections at the same time? Can I be a trustee of another election after one finishes?

Reply:
You can be a `trustee` for several `elections` at the same time. The relation between `trustees` and `elections` is encapsulated from adding a `user` as `trustee`. A `trustee` has to create the `public_keys` only once per `authority`. That's when the unique name gets added and validated. After that, this value won't get changed. No matter if the `trustee` will be "responsible" for one or many `elections`.

Reply:
@oriolgual I removed the `index` for the `name` in general and added validations for the `unique_name` in the forms: the first time, when a `user` gets added as a `trustee` and the 2nd time, when the `trustee` generates the `public key`. These validations include the `organization`. On Decidim side, the `trustee.name` won't have the `organization.name` included, but when setting up an `election`, the `trustee` data will send the `trustee.full_name`, where the `organization.name` is included. Let me know if you have any questions regarding these changes. 😊